docs/rfcs/fix-2-to-1-spacing.md

```diff
@@ -0,0 +1,54 @@
+# RFC: Fix 2:1 spacing
+
+## Summary
+
+This document suggests a way to address issues that came up after the implementation of 2:1 spacing in vertical bar charts with string x-axis. The proposed solution introduces new props that will give users more control over both bar width and spacing within the chart.
+
+## Background
+
+The 2:1 spacing feature was added through [Pull Request #25838](https://github.com/microsoft/fluentui/pull/25838), as per [the design](<https://www.figma.com/file/WOoCs0CmNYZhYl9xXeCGpi/Data-viz-(Archive)?type=design&node-id=21153-80245&mode=design&t=yuXjQl2xGWzQrsK8-4>). Following its release in version `5.16.0`, queries from partner teams arose regarding the possibility of disabling or overriding this feature, as they found the previous uniform spacing better suited to their requirements.
+
+## Problem
+
+![](../assets/images/2-to-1-spacing-problem.png)
+
+- The bars don’t occupy the entire space due to the default bar width of 16px and a fixed 2:1 spacing, often leaving considerable amount of blank space before the first bar and after the last bar. This becomes particularly noticeable when the chart width is large.
+- Additionally, this setup causes unnecessary overlapping of x-axis labels.
+
+## Proposal
+
+<iframe allowfullscreen frameborder="0" width="640" height="480" src="https://lucid.app/documents/embedded/86f214c8-0d6d-4640-a81a-fa492851f6d9" id="iv-eZ8WS-fIH"></iframe>
```

Comment:
Iframe with cross domain content is not a good security practice. It is not rendering in the browser also.

Reply:
Replaced it with image linked to diagram file